sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/consumer/strategies/synccommit/SyncCommitMLS.java

```diff
@@ -32,8 +43,12 @@ public void processRecords(ConsumerRecords<K, V> records, KafkaConsumer<K, V> co
   private void commitSync(KafkaConsumer<K, V> consumer) {
     try {
       consumer.commitSync();
-    } catch (CommitFailedException e) {
-      LOGGER.error("Commit failed", e);
+    } catch (KafkaException exception) {
+      LOGGER.error("Commit failed", exception);
+
+      if (commitSyncErrorHandler.handleError(exception, consumer)) {
+        commitSync(consumer);
+      }
```

Comment:
I would leave it to the consumers what they want to do if the commit fails.
What would you do if it fails again?